pyproject.toml

```diff
@@ -48,7 +48,7 @@ include = [
 zenml = "zenml.cli.cli:cli"
 
 [tool.poetry.dependencies]
-python = ">=3.6.2,<3.9"
+python = ">=3.8.0,<3.9"
```

Comment:
Not sure if this is a change you made or that's coming from the fuckup of all PR's, but we probably want this to stay python 3.6 for now right?

Reply:
oh crap you are ri ght, that snuck ino to my commit. will fix it